### PR TITLE
Fix/media issues notification level

### PR DESCRIPTION
**Pull request by nytamin**
This PR does some refactoring of the mediaStatus-generation and introduces some changes:

* Added `PieceStatusCode.SOURCE_HAS_ISSUES` and uses that for the "black/freeze frames detected" notifications instead of the previous `PieceStatusCode.SOURCE_BROKEN`.

This affects the GUI, by reducing "black/freeze frames detected" notification-levels to `Notes`, instead of `Warnings`.

The motivation for this is to reduce the amount of warnings for the end-users.

### CODE REVIEW

File: meteor/client/lib/notifications/notifications.ts [hunk 1]
```diff
@@ -477,13 +477,20 @@ export class Notification extends EventEmitter {
...
 
 export function getNoticeLevelForPieceStatus(statusCode: RundownAPI.PieceStatusCode): NoticeLevel | null {
-	return statusCode !== RundownAPI.PieceStatusCode.OK && statusCode !== RundownAPI.PieceStatusCode.UNKNOWN
-		? statusCode === RundownAPI.PieceStatusCode.SOURCE_NOT_SET
-			? NoticeLevel.CRITICAL
-			: // : innerPiece.status === RundownAPI.PieceStatusCode.SOURCE_MISSING ||
-			  // innerPiece.status === RundownAPI.PieceStatusCode.SOURCE_BROKEN
-			  NoticeLevel.WARNING
-		: null
+	if (statusCode === RundownAPI.PieceStatusCode.OK || statusCode === RundownAPI.PieceStatusCode.UNKNOWN) {
```
Comment (LindvedKrvang) [inline meteor/client/lib/notifications/notifications.ts:480]: Why not use a switch?

File: meteor/client/lib/notifications/notifications.ts [hunk 2]
```diff
@@ -477,13 +477,20 @@ export class Notification extends EventEmitter {
...
+		return null
```
Comment (LindvedKrvang) [inline meteor/client/lib/notifications/notifications.ts:481]: Returning null is generally a bad practice, but I see that the code allready did that before the refactor.

Comment (nytamin) [reply meteor/client/lib/notifications/notifications.ts:481]: yes, I changed as little as possible with the refactor.

File: meteor/client/lib/notifications/notifications.ts (hunk 1 above)
Comment (nytamin) [reply meteor/client/lib/notifications/notifications.ts:480]: Good point! I'll switchify it!

**Review by jstarpl — APPROVED** [review]
Looks good.

**Pull request merged**